decidim-core/app/views/layouts/decidim/shared/_layout_user_profile.html.erb

```diff
@@ -29,3 +29,17 @@
     </section>
   </div>
 <% end %>
+<script>
```

Comment:
@Stef-Rousset , can you point where is this being referenced in the provided Audit reports? 

Looking using the WGAC tool provided by Decidim, the user menu is having issues: 
<img width="826" height="450" alt="image" src="https://github.com/user-attachments/assets/4a0c9969-7cdb-4e30-a880-48992060981c" />

Reply:
hello @alecslupu , thanks for your review ! I made a mistake, this js code wasn't related to this PR, I deleted it.

Reply:
Apparently there is no mistake with this codebase. Seems that the code path is failing now ... but the issue, seems related to the one in #14951.